clickhouse_plantuml/__main__.py

```diff
@@ -173,7 +179,7 @@ def main():
     client = Client(
         host=args.host, port=args.port, user=args.user, password=args.password
     )
-    tables = Tables(client, args.databases, args.tables)
+    tables = Tables(client, args.databases, args.tables, args.merge_matviews)
```

Comment:
I'd prefer to make `Tables._merge_matviews` public and execute it in `main()` conditionally.

Reply:
I don't get it 🤓
`Tables` class doesn't have `_merge_matviews` prop. `merge_matviews` is just args prop